classes/04class/exercises/c04-iac04/carineoliveira/terraform-code/iac-04-module/load-balancer.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_alb" "my_load_balancer" {

```

Comment:
Hi Carine, as we discussed, for invoking this module from your main code, you can create a new folder per module.

In this example, let's say a new folder called `my_load_balancer_module`.  Inside this folder you'll have basically variables, resources and outputs. The resource file looks exactly as this one here. The variables file can have `subnets` and `security_groups` as input.

The part missing is calling this module from the main code. For doing that you just need to declare `module` in a file (like main.tf):

```terraform
module "my_load_balancer" {
    source = "./iac-04-module/my_load_balancer_module""

    # VARIABLES GO HERE, like this one
    subnets = data.aws_subnet_ids.my_subnet_ids.ids
}
```